browser/brave_content_browser_client.cc

```diff
@@ -271,6 +298,17 @@ void BraveContentBrowserClient::BrowserURLHandlerCreated(
   ChromeContentBrowserClient::BrowserURLHandlerCreated(handler);
 }
 
+void BraveContentBrowserClient::RenderProcessWillLaunch(
+    content::RenderProcessHost* host) {
+  Profile* profile = Profile::FromBrowserContext(
+      chrome::GetBrowserContextRedirectedInIncognito(
```

Comment:
this needs to go in `BraveRendererUpdaterFactory::GetBrowserContextToUse` so all calls to `BraveRendererUpdaterFactory::GetForProfile` return the same thing

Reply:
done